config/environments/development.rb

```diff
@@ -61,4 +61,14 @@
   # Use an evented file watcher to asynchronously detect changes in source code,
   # routes, locales, etc. This feature depends on the listen gem.
   config.file_watcher = ActiveSupport::EventedFileUpdateChecker
+
+  config.action_mailer.delivery_method = :smtp
+  config.action_mailer.smtp_settings = {
+    :user_name => JedlRails::Application.secrets.mail_user_name,
```

Comment:
Use `Rails.application.secrets` and ruby 1.9+'s hash of symbol heys syntax: `key: value` instead of `:key => value`